charts/gardener-extension-admission-aws/charts/runtime/templates/vpa.yaml

```diff
@@ -9,8 +9,7 @@ spec:
   resourcePolicy:
     containerPolicies:
     - containerName: '*'
-      minAllowed:
-        memory: {{ required ".Values.global.vpa.resourcePolicy.minAllowed.memory is required" .Values.global.vpa.resourcePolicy.minAllowed.memory }}
+      controlledValues: RequestsOnly
```

Comment:
Empty sets are falsy. I am not sure what the actual desired outcome is (@kon-angelo) but with this we get the blanket policy when we configure _anything_ in `Values.global.vpa.resourcePolicy` and nothing otherwise.
Did not catch that on the original PR :/.

Reply:
Agree, we probably want to remove all vpa configs from values.yaml and also remove the check `{{- if .Values.global.vpa.resourcePolicy }}` in vpa.yaml.

Reply:
I agree we need to remove the conditional